en/development/errors.rst

```diff
@@ -51,7 +51,7 @@ message, file and line (``debug`` enabled).
 Changing Exception Handling
 ===========================
 
-Exception handling offers 3 ways to tailor how exceptions are handled.  Each
+Exception handling offers 4 ways to tailor how exceptions are handled.  Each
```

Comment:
How about `multiple` or many etc? :) This way it will also be true if these docs ever change again.